src/Agent/NewRelic/Agent/Core/DataTransport/ConnectionHandler.cs

```diff
@@ -30,10 +30,10 @@ public class ConnectionHandler : ConfigurationBasedService, IConnectionHandler
     {
         private static readonly Dictionary<string, Action<string>> ServerLogLevelMap = new Dictionary<string, Action<string>>
         {
-            {"INFO", Log.Info},
-            {"WARN", Log.Warn},
-            {"ERROR", Log.Error},
-            {"VERBOSE", Log.Finest}
+            {"INFO", (s) => Log.Info(s)},
+            {"WARN", (s) => Log.Warn(s)},
+            {"ERROR", (s) => Log.Error(s)},
+            {"VERBOSE", (s) => Log.Finest(s)}
```

Comment:
Is there another way to reference the correct overload so that we don't have to allocate a lambda?

Reply:
Done